addons/l10n_din5008_expense/models/hr_expense_sheet.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
```

Comment:
A < Z < _ < a < z :wink:

```suggestion
from odoo import _, fields, models
```